### PR TITLE
Refs #11009 - Added description on foreman-debug.

### PR DESCRIPTION
**Pull request by alongoldboim**

### CODE REVIEW

File: _includes/manuals/1.9/7.3_getting_help.md [hunk 1]
```diff
@@ -8,3 +8,12 @@ Mailing lists are available via Google Groups. Much like IRC, we have a general
 
 * [foreman-users](https://groups.google.com/forum/?fromgroups#!forum/foreman-users)
 * [foreman-dev](https://groups.google.com/forum/?fromgroups#!forum/foreman-dev)
+
+### Gathering information
+* In order to troubleshoot and get relevant data use Foreman-debug specified in [genral support] (http://theforeman.org/support.html)
```
Comment (domcleal) [inline _includes/manuals/1.9/7.3_getting_help.md]: Remove the bullet points, these are sentences.  "foreman-debug" is a command, so is case sensitive - lower case it.  "general" is misspelt, and the link isn't working - you have a space between the brackets.  The link should also not contain the domain, just /support.html.

File: _includes/manuals/1.9/7.3_getting_help.md [hunk 2]
```diff
@@ -8,3 +8,12 @@ Mailing lists are available via Google Groups. Much like IRC, we have a general
...
+* To upload the tarball to our public server via rsync use:
```
Comment (domcleal) [inline _includes/manuals/1.9/7.3_getting_help.md]: ditto on bullet points, no need.

File: _includes/manuals/1.9/7.3_getting_help.md [hunk 3]
```diff
@@ -8,3 +8,12 @@ Mailing lists are available via Google Groups. Much like IRC, we have a general
...
+
+        Foreman-debug -u
+
+  This is a write-only directory (readable only by Foreman core developers), please note that the rsync transmission is UNENCRYPTED.
```
Comment (domcleal) [inline _includes/manuals/1.9/7.3_getting_help.md]: Stray space at the start of the line, and I'd split this into two sentences.

Consider adding the other info about filtering that are present in support.md.

File: _includes/manuals/1.9/7.3_getting_help.md [hunk 4]
```diff
@@ -8,3 +8,12 @@ Mailing lists are available via Google Groups. Much like IRC, we have a general
 
 * [foreman-users](https://groups.google.com/forum/?fromgroups#!forum/foreman-users)
 * [foreman-dev](https://groups.google.com/forum/?fromgroups#!forum/foreman-dev)
+
+### Gathering information
+* In order to troubleshoot and get relevant data use Foreman-debug specified in [genral support] (http://theforeman.org/support.html)
+* To upload the tarball to our public server via rsync use:
+
+        Foreman-debug -u
```
Comment (domcleal) [inline _includes/manuals/1.9/7.3_getting_help.md]: Some stray spaces here, when it should be four for code.

Comment (domcleal) [inline _includes/manuals/1.9/7.3_getting_help.md]: Plus case-sensitive again.

**Comment by lzap** [comment]
It is worth mentioning that the script tries to filter out generic passwords/tokens etc from configuration files, so these are not transmitted.

**Comment by domcleal** [comment]
Indeed, I'd actually _move_ this content if possible, rather than duplicating it all.  The reference to run foreman-debug according to the support page instructions is perhaps peculiar?

**Comment by alongoldboim** [comment]
@lzap your right, ill add it. 
@domcleal ill take off the reference to the support, but i do think it would be better if everything was in one place and we should place it all either in the support section or in the manual.

File: support.md [hunk 5]
```diff
@@ -10,20 +10,7 @@ title: Support
...
-    # foreman-debug
-
-If you installed from git, you can find it in the Foreman directory:
-
-    # script/foreman-debug
-
-If you run it without any options, it will collect data, filter out possible
-passwords or tokens and create a tarball which can be safely handed over to
-us.
+foreman-debug tool. For more info use the [manual](/manuals//index.html#7.2GettingHelp).
```
Comment (alongoldboim) [inline support.md]: What link should i use that will always go to the latests manual version?

Comment (stbenjam) [inline support.md]: `/manuals/latest/index.html#7.2GettingHelp`

(Note - Only works in production)

Comment (alongoldboim) [inline support.md]: @stbenjam thanks

File: support.md [hunk 6]
```diff
@@ -10,20 +10,7 @@ title: Support
...
-    # foreman-debug
-
-If you installed from git, you can find it in the Foreman directory:
-
-    # script/foreman-debug
-
-If you run it without any options, it will collect data, filter out possible
-passwords or tokens and create a tarball which can be safely handed over to
-us.
+foreman-debug tool. For more info use the [/manuals/latest/index.html#7.3GettingHelp).
```
Comment (domcleal) [inline support.md]: Missing text for this link.

**Comment by alongoldboim** [comment]
@domcleal Is it ok now or more changes needed?

File: _includes/manuals/1.9/7.3_getting_help.md [hunk 7]
```diff
@@ -8,3 +8,24 @@ Mailing lists are available via Google Groups. Much like IRC, we have a general
...
+
+    # script/foreman-debug
+
+If you run it without any options, it will collect data, filter out possible
+passwords or tokens and create a tarball which can be safely handed over to
+us.
+
+To upload the tarball to our public server via rsync use:
+
+        foreman-debug -u
```
Comment (domcleal) [inline _includes/manuals/1.9/7.3_getting_help.md]: NItpick, please match indentation of the commands above (four spaces) and add the `#` to indicate a root prompt.

**Comment by domcleal** [comment]
Thanks @alongoldboim , merged with one small tweak to the language on the support page.

**Pull request closed without merging**